composer.json

```diff
@@ -23,6 +23,7 @@
     "require-dev": {
         "erusev/parsedown": "~1.0",
         "jgm/CommonMark": "0.20",
+        "jgm/SmartPunct": "0.20",
```

Comment:
> Why? The smart_punct.txt file is included in jgm/CommonMark.

Is it? `jgm/CommonMark` points to <http://spec.commonmark.org/0.20/spec.txt> in [`composer.json`](https://github.com/thephpleague/commonmark/blob/master/composer.json#L36), which doesn't seem to include any smart punctuation tests. Additionally, there is no `smart_punct.txt` file in `jgm/CommonMark` anywhere I can find it.

There is, however a [`smart_punct.txt` file in `jgm/CommonMark.js`](https://github.com/jgm/commonmark.js/blob/master/test/smart_punct.txt), which is what `jgm/SmartPunct` points to in this file.